reporters/reporter.go

```diff
@@ -0,0 +1,61 @@
+// maestro
+// https://github.com/topfreegames/maestro
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package reporters
+
+import (
+	"sync"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+type Reporter interface {
```

Comment:
Before every public function, method, interface and struct we comment a brief description about it. For example,

//Reporter is the reporter interface